file-formats/rvbc/type/zif_aff_rvbc_v1.intf.abap

```diff
@@ -0,0 +1,102 @@
+"! AFF type for the object RVBC
+INTERFACE zif_aff_rvbc_v1
+  PUBLIC.
+  "! $values { @link zif_aff_rvbc_v1.data:co_extensibility_mode }
+  "! $default { @link zif_aff_rvbc_v1.data:co_extensibility_mode.not_supported }
+  TYPES ty_extensibility_mode    TYPE n LENGTH 2.
+  TYPES ty_app_id                TYPE c LENGTH 70.
+  TYPES ty_source                TYPE n LENGTH 1.
+  TYPES ty_status                TYPE n LENGTH 1.
+  TYPES ty_constcy_status        TYPE n LENGTH 1.
+  TYPES ty_abap_language_version TYPE c LENGTH 1.
+  TYPES ty_ina1_service_id       TYPE c LENGTH 30.
+  TYPES ty_ina1_service_name     TYPE c LENGTH 40.
+  TYPES ty_ina1_service_type     TYPE n LENGTH 2.
+  TYPES ty_main_language         TYPE c LENGTH 1.
+
+  CONSTANTS:
+    BEGIN OF co_extensibility_mode,
+      "! <p class="shorttext">Fully supported include custom analytical queries</p>
```

Comment:
title case: in titles all words are capitalised

Reply:
changed accordingly in the latest commit